client-src/elements/form-field-specs.js

```diff
@@ -77,54 +83,101 @@ const OT_MILESTONE_WEBVIEW_RANGE = {
   later: 'ot_milestone_webview_end',
 };
 
-export const OT_SHIPPED_MILESTONE_DESKTOP_RANGE = {
+const OT_ALL_SHIPPED_MILESTONE_DESKTOP_RANGE = {
   earlier: 'ot_milestone_desktop_start',
+  allLater: 'shipped_milestone',
+  warning: 'Origin trial should start before feature is shipped.',
+};
+
+const ALL_OT_SHIPPED_MILESTONE_DESKTOP_RANGE = {
+  allEarlier: 'ot_milestone_desktop_start',
   later: 'shipped_milestone',
-  error: 'Origin trial must start before feature is shipped.',
+  warning: 'All origin trials should start before feature is shipped.',
 };
 
-const OT_SHIPPED_MILESTONE_WEBVIEW_RANGE = {
+const OT_ALL_SHIPPED_MILESTONE_WEBVIEW_RANGE = {
   earlier: 'ot_milestone_webview_start',
+  allLater: 'shipped_webview_milestone',
+  warning: 'Origin trial should start before feature is shipped.',
+};
+
+const ALL_OT_SHIPPED_MILESTONE_WEBVIEW_RANGE = {
+  allEarlier: 'ot_milestone_webview_start',
   later: 'shipped_webview_milestone',
-  error: 'Origin trial must start before feature is shipped.',
+  warning: 'Origin trial should start before feature is shipped.',
 };
 
-const OT_SHIPPED_MILESTONE_ANDROID_RANGE = {
+const OT_ALL_SHIPPED_MILESTONE_ANDROID_RANGE = {
   earlier: 'ot_milestone_android_start',
-  later: 'shipped_android_milestone',
-  error: 'Origin trial must start before feature is shipped.',
+  allLater: 'shipped_android_milestone',
+  warning: 'Origin trial should start before feature is shipped.',
 };
 
-const OT_SHIPPED_MILESTONE_IOS_RANGE = {
-  earlier: 'ot_milestone_ios_start',
-  later: 'shipped_ios_milestone',
-  error: 'Origin trial must start before feature is shipped.',
+const ALL_OT_SHIPPED_MILESTONE_ANDROID_RANGE = {
+  allEarlier: 'ot_milestone_android_start',
+  later: 'shipped_android_milestone',
+  warning: 'Origin trial should start before feature is shipped.',
 };
 
-const DT_SHIPPED_MILESTONE_DESKTOP_RANGE = {
+// No ot_milestone_ios_start or _end, yet.
+// const OT_ALL_SHIPPED_MILESTONE_IOS_RANGE = {
+//   earlier: 'ot_milestone_ios_start',
+//   allLater: 'shipped_ios_milestone',
+//   warning: 'Origin trial should start before feature is shipped.',
+// };
+// const ALL_OT_SHIPPED_MILESTONE_IOS_RANGE = {
+//   allEarlier: 'ot_milestone_ios_start',
+//   later: 'shipped_ios_milestone',
+//   warning: 'Origin trial should start before feature is shipped.',
+// };
+
+const DT_ALL_SHIPPED_MILESTONE_DESKTOP_RANGE = {
   earlier: 'dt_milestone_desktop_start',
+  allLater: 'shipped_milestone',
+  warning: 'Shipped milestone must be later than dev trial.',
+};
+
+const ALL_DT_SHIPPED_MILESTONE_DESKTOP_RANGE = {
+  allEarlier: 'dt_milestone_desktop_start',
   later: 'shipped_milestone',
-  error: 'Shipped milestone must be later than dev trial.',
+  warning: 'Shipped milestone must be later than dev trial.',
```

Comment:
To match other wording
```suggestion
  warning: 'Shipped milestone should be later than dev trial start milestone.',
```

Reply:
done